tests/unit/test_charm.py

```diff
@@ -215,3 +215,7 @@ def test_charm_removes_job_when_empty_targets_are_specified(self):
 
         self.assertIsInstance(self.harness.model.unit.status, BlockedStatus)
         self.assertEqual({"scrape_jobs": "[]"}, dict(relation_data))
+
+    def test_workload_version_is_set(self):
+        self.harness.charm._on_install(None)
```

Comment:
Can we make this slightly more blackboxish?
E.g. begin with initial hooks -> assert workload version is something that isn't nothing. Would be even easier to copy-paste elsewhere.

Reply:
Done.